.changeset/fair-cats-fail.md

```diff
@@ -0,0 +1,6 @@
+---
+'@rsbuild/shared': minor
+'@rsbuild/core': minor
```

Comment:
Rsbuild is still in 0.x version, so:

- bugfix / feature -> patch
- breaking change -> minor